client/web/src/enterprise/batches/batch-spec/edit/DownloadSpecModal.story.tsx

```diff
@@ -1,14 +1,19 @@
-import { storiesOf } from '@storybook/react'
+import { DecoratorFn, Meta, Story } from '@storybook/react'
 
 import { WebStory } from '../../../../components/WebStory'
 
 import { DownloadSpecModal } from './DownloadSpecModal'
 
-const { add } = storiesOf('web/batches/batch-spec/edit', module).addDecorator(story => (
-    <div className="p-3 container">{story()}</div>
-))
+const decorator: DecoratorFn = story => <div className="p-3 container">{story()}</div>
 
-add('DownloadSpecModal', () => (
+const config: Meta = {
+    title: 'web/batches/batch-spec/edit',
+    decorators: [decorator],
+}
+
+export default config
+
+export const _DownloadSpecModal: Story = () => (
```

Comment:
Since we rename the story using the `storyName` property, let's avoid using `_` in exports and add the `Story` suffix instead.
```suggestion
export const DownloadSpecModalStory: Story = () => (
```